CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Changed
 
 - The SDK now uses logr as the default logger to unify the logging output with the controller-runtime logs. Users can still use a logger of their own choice. See the [logging doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/user/logging.md) on how the SDK initializes and uses logr.
+- `operator-sdk print-deps` prints Golang packages and versions expected by the Operator SDK version being used by an operator. Supplying `--as-file` prints packages and versions in Gopkg.toml format.
```

Comment:
`expected by the Operator SDK version being used by an operator` => `expected by the current Operator SDK version.`

Also this should go under the section `Added`.
-  A new cmd `operator-sdk print-deps` which prints ...

And we can link to the [CLI reference guide](https://github.com/operator-framework/operator-sdk/blob/master/doc/sdk-cli-reference.md#print-deps) as well